swap_meet/clothing.py

```diff
@@ -1,2 +1,9 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+class Clothing(Item):
+    
+    def __init__(self, category="Clothing", condition=0):
+        self.category = category
+        self.condition = condition
```

Comment:
Since you want every instance to have a category of clothing it's better if it isn't a parameter:

```suggestion
    def __init__(self, condition=0):
        self.category = "Clothing"
        self.condition = condition
```